app/src/main/java/zapsolutions/zap/connection/manageWalletConfigs/WalletConfigsJson.java

```diff
@@ -11,7 +11,17 @@
     @SerializedName("connections")
     List<WalletConfig> mConnections;
```

Comment:
Wondering if we should use a `Set` here, as we don't want to store duplicated or `null` values.